### ISSUE TITLE
[Bug]: Delete DNS record example is wrong in documentation

### ISSUE THREAD

**Issue by maximouche** [linode/ansible_linode#462]
### Ansible Version

2.15.5

### Ansible Configuration

NA

### Effected Components

Documentation (https://github.com/linode/ansible_linode/blob/main/docs/modules/domain_record.md) says
```
- name: Delete a domain record
  linode.cloud.domain:
    domain: my-domain.com
    name: my-subdomain
    state: absent
```
but type and target attribute are required to properly delete a record.

### Expected Behavior

Documentation should provide a valid example to delete a dns_record

Doc should target and type for the delete example :
```
- name: Delete a domain record
  linode.cloud.domain:
    domain: my-domain.com
    name: my-subdomain
    type: 'A'
    target: '127.0.0.1'
    state: absent
```

### Actual Behavior

Current example is wrong

### Steps to Reproduce

Test the current example in the doc

**Comment by lgarber-akamai** [comment]
Thank you for the report! I've made a ticket in our internal tracker to correct the `domain_record` documentation and will update this ticket once it's resolved 🙂

**Comment by yec-akamai** [comment]
We just released the fix in [v0.25.2](https://github.com/linode/ansible_linode/releases/tag/v0.25.2). Thank you for reporting the issue!
